mobile/native/src/ln_dlc/recover_rollover.rs

```diff
@@ -1,74 +1,85 @@
 use crate::ln_dlc::node::Node;
 use anyhow::Result;
+use bitcoin::hashes::hex::ToHex;
 use ln_dlc_node::node::rust_dlc_manager::channel::signed_channel::SignedChannelState;
 use ln_dlc_node::node::rust_dlc_manager::channel::Channel;
 use ln_dlc_node::node::rust_dlc_manager::Storage;
 
 impl Node {
     /// Checks and recovers from a potential stuck rollover.
     pub async fn recover_rollover(&self) -> Result<()> {
-        tracing::debug!("Checking if dlc channel got stuck in rollover.");
+        tracing::debug!("Checking if DLC channel got stuck during rollover");
+
         let channels = self.inner.channel_manager.list_channels();
         let channel_details = match channels.first() {
             Some(channel_details) => channel_details,
             None => {
-                tracing::debug!("No channel found. All good!");
                 return Ok(());
             }
         };
 
-        let dlc_channels = self.inner.dlc_manager.get_store().get_sub_channels()?;
-        let dlc_channel = dlc_channels
+        let subchannels = self.inner.dlc_manager.get_store().get_sub_channels()?;
+        let subchannel = match subchannels
             .iter()
-            .find(|dlc_channel| dlc_channel.channel_id == channel_details.channel_id);
-
-        let dlc_channel = match dlc_channel {
-            Some(dlc_channel) => dlc_channel,
+            .find(|dlc_channel| dlc_channel.channel_id == channel_details.channel_id)
+        {
+            Some(subchannel) => subchannel,
             None => {
-                tracing::debug!("No dlc channel found. All good!");
                 return Ok(());
             }
         };
 
-        let dlc_channel_id = match dlc_channel.get_dlc_channel_id(0) {
+        let channel_id_hex = subchannel.channel_id.to_hex();
+
+        let dlc_channel_id = match subchannel.get_dlc_channel_id(0) {
             Some(dlc_channel_id) => dlc_channel_id,
             None => {
-                tracing::warn!(channel_id=%hex::encode(dlc_channel.channel_id), "Couldn't get a dlc channel id for a dlc channel");
+                tracing::debug!(
+                    channel_id=%channel_id_hex,
+                    "Cannot consider subchannel for rollover recovery without DLC channel ID"
+                );
                 return Ok(());
             }
         };
 
-        let channel = self
+        let dlc_channel_id_hex = dlc_channel_id.to_hex();
+
+        let dlc_channel = self
             .inner
             .dlc_manager
             .get_store()
             .get_channel(&dlc_channel_id)?;
 
-        let signed_channel = match channel {
+        let signed_channel = match dlc_channel {
             Some(Channel::Signed(signed_channel)) => signed_channel,
-            Some(channel) => {
-                tracing::warn!(dlc_channel_id=%hex::encode(dlc_channel_id), "Found channel in unexpected state. Expected: Signed, Found: {channel:?}");
```

Comment:
❓ why did you remove this log?

Reply:
I will reintroduce it although I think it should be on DEBUG, since there's nothing concerning about finding a channel that is not yet signed, right?